### DISCUSSION TITLE
Update/Delete members of a collection

### DISCUSSION THREAD

**Discussion by GoncaloPataca** [OData/AspNetCoreOData#1102] [Ideas]
Hi, according to the OData documentation, we should be able to use a $filter and $each parameter in the route in conjunction with the DELETE method (as well as the PATCH method) http://docs.oasis-open.org/odata/odata/v4.01/odata-v4.01-part1-protocol.html#sec_DeleteMembersofaCollection. I couldn't find anything related to this feature in this repo, is this a planned feature for the near future?

**Comment by figuerres** [comment]
OData and the idea of "REST" full web services have long had the idea that the "right way" to do an OData controller is to use a pattern
where you have a controller for an entity say "Customers"
you map HTTP GET to get a list of them and to get a single one by the key
to then map HTTP POST  to inserting one new entity
HTTP DELETE with the key to removong an entity
and HTTP PUT or PATCH for update of an entity.

that patten make for a nice match of the http protocol to the logical CRUD operations on a data store.

**Reply by GoncaloPataca** [reply]
Thank you for the explanation of the RESTful design pattern. However, my initial question was specific to the OData documentation mentioning the use of $filter and $each parameters in the DELETE method. I couldn't find information about this feature in this repository, and I'm curious if it's a planned feature in the near future. I appreciate your help!
